api/app/signals/apps/api/v1/filters.py

```diff
@@ -95,6 +105,10 @@ class SignalFilter(FilterSet):
 
     stadsdeel = filters.MultipleChoiceFilter(field_name='location__stadsdeel',
                                              choices=stadsdelen)
+    area_type = filters.MultipleChoiceFilter(field_name='location__area_type',
+                                             choices=area_type_choices)
```

Comment:
It feels a bit strange that we also can filter on the `AreaType` alone. Do we want this to be possible? We configure what type of area's we can define on a location model. I am not sure how to solve this at the moment 🤔 .

When filtering on an `area_type` the `area_code` is always needed (and vice versa).

Reply:
agreed, the general idea was to filter based on `area_type` and `area_code`. I can make the combination mandatory?

I think we should handle this properly when the 'stadsdeel', 'buurt_code' stuff will be eventually merged into area_type. We might need to define the relation at that point. (many to many: a signal could have a 'stadsdeel', 'buurt_code', 'district' relation via location)

Reply:
+1 on @vanbuiten 's remark

Regarding the relations, probably depends on the use of this field. If it is only occasionally needed, we could leave them off and just do a live geo query on Postgis. Otherwise we have to careful that the database relation matches the spatial relation, i.e. make sure we update on changes to an Area and on changes to the location).